docs/Design/Null Check.md

```diff
@@ -0,0 +1,140 @@
+# Null Check
+Creates checks for if the passed variable has value null and/or undefined (see errata). Can be used to check positvly for null or negetivly for null

```

Comment:
We shouldn't care in this general part of the spec that there is a conceptual difference between null and undefined. Instead, we should just define some sort null"-ness (not having value) and only later explain the difference, *if* we need.